demos/svg/lv_demo_svg.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file lv_demo_svg.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_demo_svg.h"
+
+#if LV_USE_DEMO_SVG
+
+extern size_t svg_len;
+extern char svg_data[];
```

Comment:
Should these symbols be namespaced a bit more? Maybe `img_demo_svg_tiger_len` and `img_demo_svg_tiger_data` to match some of the other demo assets?

Reply:
done